src/EnergyPlus/OutsideEnergySources.cc

```diff
@@ -249,7 +249,7 @@ void GetOutsideEnergySourcesInput(EnergyPlusData &state)
                 ErrorsFound = true;
             }
             if (!ScheduleManager::CheckScheduleValueMinMax(
-                    state, state.dataOutsideEnergySrcs->EnergySource(EnergySourceNum).CapFractionSchedNum, ">=", 0.0)) {
```

Comment:
No more ">=" or ">" or incorrectly ">=0".  Just use a bool to determine if it includes the "or equals" part.